### DISCUSSION TITLE
分享网站（重定向）

### DISCUSSION THREAD

**Discussion by su2569** [share121/inter-knot#1314] [其他]
[texs.e.cn.vc](http://texs.e.cn.vc/)
想的话就把跳转URL发在评论区

**Comment by su2569** [comment]
比如https://www.bilibili.com/

**Comment by lllliufeng** [comment]
> [!CAUTION]
> 即将跳转到外部网站，谨防[诈骗](https://www.bilibili.com/video/BV1GJ411x7h7/)

**Comment by su2569** [comment]
现在换了

**Comment by su2569** [comment]
真的
